docs/content-services/services/security-controls.service.md

```diff
@@ -0,0 +1,79 @@
+---
+Title: Security Controls service
+Added: v2.0.0
+Status: Active
+Last reviewed: 2023-02-21
+---
+
+# [Security Controls service](../../../lib/content-services/src/lib/security/services/security-controls-groups-marks-security.service.ts "Defined in security-controls-groups-marks-security.service.ts")
+
+Manages security groups & marks in Content Services.
+
+## Class members
+
+### Methods
+
+-   **getSecurityGroup**(skipCount?: `number`, maxItems?: `number`, include?: `string`): [`Promise`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises)`<SecurityControlsGroupResponse>`<br/>
+    Get security groups.
+    -   _skipCount:_ `number`  - The number of entities that exist in the collection before those included in this list.
+    -   _maxItems:_ `number`  - The maximum number of items to return in the list.
+    -   _include:_ `string`  - Additional information about the security group.
+    -   **Returns** [`Promise`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises)`<SecurityControlsGroupResponse>`
+-   **createSecurityGroup**(input: `SecurityGroupBody`): [`Observable`](http://reactivex.io/documentation/observable.html)`<SecurityGroupEntry>`<br/>
+    Creates a security group.
+    -   _input:_ `SecurityGroupBody`  - SecurityGroupBody
+    -   **Returns** [`Observable`](http://reactivex.io/documentation/observable.html)`<SecurityGroupEntry>` 
+-   **createSecurityMarks**(securityGroupId: `string`, input: `SecurityMarkBody[]`): [`Promise`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises)`<SecurityMarkEntry|SecurityMarkPaging>`<br/>
+    Create security marks.
+    -   _securityGroupId:_ `string`  - The key for the security group id.
+    -   _SecurityMarkBody:_ `SecurityMarkBody[]` 
+    -   **Returns** [`Promise`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises)`<SecurityMarkEntry|SecurityMarkPaging>`
+-   **getSecurityMark**(SecurityGroupId: `string`, skipCount?: `number`, include?: `string`): [`Promise`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises)`<SecurityControlsMarkResponse>`<br/>
+    Get security mark value.
+    -   _SecurityGroupId:_ `string`  - The key for the security group id.
+    -   _skipCount:_ `number`  - The number of entities that exist in the collection before those included in this list.
+    -   _include:_ `string`  - The key for the security mark is in use or not.
+    -   **Returns** [`Promise`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises)`<SecurityControlsMarkResponse>` - Null object when the operation completes
```

Comment:
That statement is incorrect probably: "Null object when the operation completes". It returns SecurityControlsMarkResponse object when completes.